docs/checks/approximateWayCheck.md

```diff
@@ -0,0 +1,10 @@
+# ApproximateWayCheck
+
+#### Description
```

Comment:
Could you add a section to include examples? You can refer any of the existing docs for the details. Thanks!